### PR TITLE
Feat(BILL-CPC-648...): Implemented all methods to inventory service

### PR DESCRIPTION
**Pull request by ElricVermaut**
[JIRA: link to jira ticket](https://champlainsaintlambert.atlassian.net/browse/CPC-648?atlOrigin=eyJpIjoiYWNjMGFhYTg0NGRiNDk2N2JjZmIxZGRmZTc2YjkwMDgiLCJwIjoiaiJ9)
## Context:
Added all methods and necessary files to inventory service
## Changes
- Added getBundleByBundleUUID
- Added getAllBundles
- Added createBundle
- Added deleteBundle
- Added getBundlesByItem
- Additional file changes to implement inventory-service

### CODE REVIEW

**Review by Marina1411 — DISMISSED** [review]

File: inventory-service/build.gradle [hunk 1]
```diff
@@ -26,8 +26,41 @@ dependencies {
     testImplementation 'org.springframework.boot:spring-boot-starter-test'
     testImplementation 'de.flapdoodle.embed:de.flapdoodle.embed.mongo'
     testImplementation 'io.projectreactor:reactor-test'
+    implementation 'org.springframework.boot:spring-boot-starter-validation'
 }
 
 jacoco {
     toolVersion = "0.8.8"
 }
+/**
```
Comment (swafit) [inline inventory-service/build.gradle:35]: Please elaborate as to why you have removed the jacoco testing from this new micro-service

File: inventory-service/src/main/java/com/petclinic/inventoryservice/datalayer/Bundle.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+import java.util.UUID;
+
+@Data
+@ToString
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class Bundle {
+
+    @Id
```
Comment (swafit) [inline inventory-service/src/main/java/com/petclinic/inventoryservice/datalayer/Bundle.java:17]: the id annotation defined here is not properly used. You should assign this to a parameter id which will be the privately stored id and the bundleUUID should be the one being accessed and not be annotated with @Id

**Review by swafit — COMMENTED** [review]
Please review the comments left. Good job on the conception of this new micro-service.

File: inventory-service/build.gradle (hunk 1 above)
Comment (ElricVermaut) [reply inventory-service/build.gradle:35]: because the testing is not implemented yet

**Review by swafit — APPROVED** [review]

**Review by Marina1411 — APPROVED** [review]

**Review by C-hunker — APPROVED** [review]

**Pull request closed without merging**